<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PasswordDialog</class>
 <widget class="QDialog" name="PasswordDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>193</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="focusPolicy">
   <enum>Qt::TabFocus</enum>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>open encrypted volume</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QPushButton" name="PushButtonOpen">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>160</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;open</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="PushButtonCancel">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>160</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;cancel</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxReadOnly">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>open the volume in &amp;read only mode</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="PushButtonMountPointPath">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>40</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>select mount point path</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="PushButtonVolumePath">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>10</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>open volume path</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="PassPhraseField">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>130</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>100</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonPassPhraseFromFile">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>130</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>open key file</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="OpenVolumePath">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="MountPointPath">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelPassphrase">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>130</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>key</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelMoutPointPath">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>mount name</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelVolumePath">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>volume path</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonPlugin">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>100</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pbKeyOption">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>130</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="cbKeyType">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>100</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>key</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>keyfile</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>plugin</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>TrueCrypt keys</string>
    </property>
   </item>
  </widget>
 </widget>
 <tabstops>
  <tabstop>OpenVolumePath</tabstop>
  <tabstop>PushButtonVolumePath</tabstop>
  <tabstop>MountPointPath</tabstop>
  <tabstop>PushButtonMountPointPath</tabstop>
  <tabstop>checkBoxReadOnly</tabstop>
  <tabstop>cbKeyType</tabstop>
  <tabstop>pushButtonPlugin</tabstop>
  <tabstop>PassPhraseField</tabstop>
  <tabstop>pbKeyOption</tabstop>
  <tabstop>pushButtonPassPhraseFromFile</tabstop>
  <tabstop>PushButtonOpen</tabstop>
  <tabstop>PushButtonCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
